ui/src/pages/index.tsx

```diff
@@ -78,6 +78,14 @@ const Home: Types.NextPage<Props> = (props): React.ReactElement => {
                                     Learn more
                                 </span>
                             </Components.Link>
+                            <Components.Link
```

Comment:
elements need to wrap at some point
![image](https://user-images.githubusercontent.com/48569671/229827026-4c7a7217-7d60-47f3-9acd-2cf3eff2c07e.png)

Reply:
this is still broken
![image](https://user-images.githubusercontent.com/48569671/230003761-6e67b001-421c-4a91-b53e-c4b99a5155b2.png)

buttons and search should be aligned in some way, maybe something similar to this
![image](https://user-images.githubusercontent.com/48569671/230007522-2439a906-1cb2-41bf-a41c-ff755cf683f5.png)

Reply:
it still looks misaligned, and the gap between 'Learn more' and 'Author Guide' is too big. The buttons also wrap too early.
![image](https://user-images.githubusercontent.com/48569671/230075998-55e7f184-de32-42b1-981b-c0f6845c3d38.png)
![image](https://user-images.githubusercontent.com/48569671/230076310-7c0ae0f6-c95e-42a5-9975-3cc0d0b4e56d.png)

Reply:
Maybe, but "too big" is subjective, I personally think that this design looks nicer. I'm not really sure how to resolve this, other than asking Tim, since it's just opinion over opinion, then again that would be another opinion, as long as it functions without "breaking" anything, like it did in the original, I'd say is acceptable ?

![54565856f3175887a0bc424aa39af989](https://user-images.githubusercontent.com/69204118/230091253-9e0c90cb-fb0e-4bb8-a8db-b0fa449ac50d.png)

![c1d7eaeca2a251312d133c8954727a66](https://user-images.githubusercontent.com/69204118/230092041-d573c37f-cb0a-4c44-a0bd-749a358c8ac6.png)